chains/celo/listener.go

```diff
@@ -33,3 +40,21 @@ func (l *listener) start() error {
 func (l *listener) close() {
 	l.conn.Close()
 }
+
+func (l *listener) getTransactionBlockHash(hash common.Hash) (tx *types.Transaction) {
+	client := l.conn.Client()
+	blockHash, _, err := client.TransactionByHash(context.Background(), hash)
+	if err != nil {
+		fmt.Errorf("unable to get BlockHash: %s", err)
+	}
+	return blockHash
```

Comment:
So this seems to imply that we're retrieving the block hash, when we're retrieving and returning the entire transaction object.